pkg/api/admin/openshiftcluster.go

```diff
@@ -82,13 +82,29 @@ const (
 	FipsValidatedModulesDisabled FipsValidatedModules = "Disabled"
 )
 
+// MaintenanceState represents the maintenance state of a cluster.
+// This is used by cluster monitornig stack to emit maintenance signals to customers.
+type MaintenanceState string
+
+const (
+	MaintenanceStateNone      MaintenanceState = "None"
+	MaintenanceStatePending   MaintenanceState = "Pending"
+	MaintenanceStatePlanned   MaintenanceState = "Planned"
+	MaintenanceStateUnplanned MaintenanceState = "Unplanned"
+)
+
 type MaintenanceTask string
 
 const (
-	MaintenanceTaskEverything  MaintenanceTask = "Everything"
-	MaintenanceTaskOperator    MaintenanceTask = "OperatorUpdate"
-	MaintenanceTaskRenewCerts  MaintenanceTask = "CertificatesRenewal"
-	MaintenanceTaskPucmPending MaintenanceTask = "PucmPending"
+	MaintenanceTaskEverything MaintenanceTask = "Everything"
+	MaintenanceTaskOperator   MaintenanceTask = "OperatorUpdate"
+	MaintenanceTaskRenewCerts MaintenanceTask = "CertificatesRenewal"
+
+	// Maintenance tasks for updating customer maintenance signals
+	// None signal should only be used when (1) admin update fails and (2) SRE fixes the failed admin update without running another admin updates
+	// Admin update success should automatically set the cluster into None state
+	MaintenanceTaskPending MaintenanceTask = "PucmPending"
```

Comment:
Is it possible to change this to `MaintenancePending`?

Reply:
We can, but I'll need to do a corresponding code change in our PowerShell script: [link](https://msazure.visualstudio.com/AzureRedHatOpenShift/_git/ARO-Scripts?path=/powerShellActions/pucm_pending.ps1&version=GBmaster&line=157&lineEnd=158&lineStartColumn=1&lineEndColumn=1&lineStyle=plain&_a=contents)

Since we need to do an RP release to deploy this code change, we'll need to coordinate code changes as we do maintenance prior to an RP release.  I say we:
1. Change the maintenance task name to `MaintenancePending` as you suggested.
2. We'll do maintenance pending + RP release.
3. After the next RP release, I'll create a PR to update the PowerShell script.

Thoughts?